src/Visualization/Utility/SelectionPolygon.cpp

```diff
@@ -127,6 +128,23 @@ std::shared_ptr<PointCloud> SelectionPolygon::CropPointCloud(
     }
 }
 
+std::shared_ptr<TriangleMesh> SelectionPolygon::CropTriangleMesh(
```

Comment:
Should these functions be exposed in Python too?

Reply:
done.